spy/tests/test_llwasm.py

```diff
@@ -1,32 +1,93 @@
 import pytest
-from spy.llwasm import LLWasmModule, LLWasmInstance, HostModule
+from spy import ROOT
 from spy.tests.support import CTest
+from spy.cbuild import EmscriptenToolchain
+from pytest_pyodide import run_in_pyodide
 
+PYODIDE = ROOT.join('..', 'node_modules', 'pyodide')
+HAS_PYODIDE = PYODIDE.check(exists=True)
+
+@pytest.mark.usefixtures('init_llwasm')
 class TestLLWasm(CTest):
 
+    @pytest.fixture(params=[
+        # "normal" execution, under CPython
+        pytest.param('wasmtime', marks=pytest.mark.wasmtime),
+
+        # run tests inside pyodide, using the 'emscripten' llwasm backend
+        pytest.param('pyodide', marks=[
+            pytest.mark.pyodide,
+            pytest.mark.skipif(
+                not HAS_PYODIDE,
+                reason="pyodide not found, run npm i")
+        ]),
+    ])
+    def llwasm_backend(self, request):
+        return request.param
+
+    @pytest.fixture
+    def init_llwasm(self, request, llwasm_backend, runtime):
+        # XXX: the "runtime" fixture is a temporary hack which hopefully we'll
+        # be able to remove soon.
+        #
+        # It is provided by pytest_pyodide and its valude is "node". Ideally,
+        # we would like to request it ONLY when llwasm_backend=='pyodide', but
+        # we couldn't find a way to do that.
+        #
+        # If we don't require "runtime" in the function signature, the
+        # "getfixturevalue('selenium')" below fails with the following
+        # message:
+        # The requested fixture has no parameter defined for test:
+        #     spy/tests/test_llwasm.py::TestLLWasm::test_call[pyodide]
```

Comment:
@hoodmane do you have an idea what causes this?
For more context, `init_llwasm` is a fixture which is automatically used by all tests in `TestLLWasm`. Ideally, for each of these tests I'd like two versions:
1. `test_xxx[wasmtime]`
2. `test_xxx[node-pyodide]`

Note that the `runtime` fixture is completely unused in the `[wasmtime]` case, but try to comment it out and see the error that you get.

It's not a big deal, but it's a bit annoying because now I get the `[node-*]` prefix also in wasmtime tests, which looks a bit confusing:
  - `test_xxx[node-wasmtime]`